test-api-generator/citrus-test-api-generator-core/src/main/java/org/citrusframework/openapi/generator/JavaCitrusCodegen.java

```diff
@@ -0,0 +1,345 @@
+package org.citrusframework.openapi.generator;
+
+import static java.util.Arrays.asList;
+import static java.util.stream.Collectors.toMap;
+import static org.openapitools.codegen.CliOption.newString;
+
+import io.swagger.v3.oas.models.OpenAPI;
+import io.swagger.v3.oas.models.info.Info;
+import java.io.File;
+import java.util.ArrayList;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.Set;
+import org.openapitools.codegen.CodegenType;
+import org.openapitools.codegen.SupportingFile;
+import org.openapitools.codegen.languages.AbstractJavaCodegen;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * @author Thorsten Schlathoelter
+ */
```

Comment:
```suggestion
```

personally, I don't like these `@author` comments.. that's all part of the git history. but maybe you (or @christophd) have other opinions on that matter.

anyway, all your files are missing the copyright comment on top of them. I think this is especially important for java source files, not necessarily for other types.

Reply:
+1 on skipping the `@author` information as this is really in the github history.

Adding the copyright license information to all source files is more important